op-service/eth/types.go

```diff
@@ -425,6 +425,8 @@ type PayloadAttributes struct {
 	GasLimit *Uint64Quantity `json:"gasLimit,omitempty"`
 	// EIP-1559 parameters, to be specified only post-Holocene
 	EIP1559Params *Bytes8 `json:"eip1559Params,omitempty"`
+	// MinBaseFeeLog2 is the minimum base fee log2.
+	MinBaseFeeLog2 uint8 `json:"minBaseFeeLog2,omitempty"`
```

Comment:
probably don't need in `PayloadAttributes` and only `SystemConfig`?

Reply:
yes, it's also needed in `AttributesMatchBlock`